pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -548,6 +548,22 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 							operatorHealthImpactLabelKey: "critical",
 						},
 					},
+					{
+						Record: "kubevirt_api_request_deprecated_total",
+						Expr:   intstr.FromString("apiserver_requested_deprecated_apis{group=\"kubevirt.io\"} * on(group,version,resource,subresource) group_right() apiserver_request_total"),
+					},
+					{
+						Alert: "KubeVirtDeprecatedAPIsRequested",
+						Expr:  intstr.FromString("sum(kubevirt_api_request_deprecated_total) > 0"),
```

Comment:
Please use the `increase` function here, so that the alert will be cleared after the defined time. 
Example: 
```
sum(increase(kubevirt_api_request_deprecated_total[5m]) )> 0
```
And please update the tests to check its being cleared after the defined period.